tests/general/IndentationBearTest.py

```diff
@@ -151,3 +155,14 @@ def test_unmatched_indents(self):
 
         invalid_file2 = ('{}}\n',)
         self.verify_bear(valid_file=None, invalid_file=invalid_file2)
+
+    def test_multiple_indent_specifiers(self):
```

Comment:
I made this line yellow, I don't know how :D

anyway, have you though about using multi-line strings like so for your test?

```python
valid file = """
multi
line
    indented
file
""".splitlines(True)[1:]
```

or maybe just do the same you are doing now but indent with spaces instead of tabs? That would make it a little easier to read which lines are indented